docs/tutorials/Aragon_Core_User_Guide.md

```diff
@@ -0,0 +1,197 @@
+## Aragon Core User Guide
+#### Documentation Version 1.0.5.1
+#### Aragon Core Version 0.5
+
+### 0. Prerequisites
```

Comment:
There shouldn't be numbering for  the titles since they are emphazised by the hashes already and isn't consistent with our other content in the wiki